src/hooks/screen/useChatScreenHook.ts

```diff
@@ -143,19 +145,16 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     return Promise.all(attachmentPromises);
   };
 
+  const sendChatSignedMutation = useSendSignedMessage();
+  const sendChatAnonMutation = useSendAnonMessage();
+
   const sendChat = async (
     message: string = randomString(20),
     attachments: [] = [],
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     iteration = 0,
     sendingChatSchema: ChatSchema | null = null
   ) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function has been refactored to use the `useSendSignedMessage` and `useSendAnonMessage` hooks for sending messages. The retry logic that was previously in the catch block of the try-catch statement has been removed. Instead, the `useMutation` hook from 'react-query' is used with the `retry: true` option to automatically retry failed mutations.
```diff
+   const sendChatSignedMutation = useSendSignedMessage();
+   const sendChatAnonMutation = useSendAnonMessage();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->